src/vcQuadTree.cpp

```diff
@@ -159,12 +160,13 @@ void vcQuadTree_InitNode(vcQuadTree *pQuadTree, uint32_t slotIndex, const udInt3
 
 bool vcQuadTree_IsNodeVisible(const vcQuadTree *pQuadTree, const vcQuadTreeNode *pNode)
 {
-  return -1 < vcQuadTree_FrustumTest(pQuadTree->frustumPlanes, udDouble3::create(pNode->tileCenter, 0.0), udDouble3::create(pNode->tileExtents, 0.0));
+  return -1 < vcQuadTree_FrustumTest(pQuadTree->frustumPlanes, udDouble3::create(pNode->tileCenter, pQuadTree->quadTreeHeightOffset), udDouble3::create(pNode->tileExtents, 0.0));
 }
 
-inline bool vcQuadTree_ShouldSubdivide(vcQuadTree *pQuadTree, double distanceMS, int depth)
+inline bool vcQuadTree_ShouldSubdivide(double distance, int depth)
 {
-  return distanceMS < (1.0 / (1 << (depth + vcTRMQToDepthModifiers[pQuadTree->pSettings->maptiles.mapQuality])));
+  // trial and error'd this heuristic
+  return distance < (10000000 >> depth);
```

Comment:
Could this value be given a meaningful name? In the interest of progressing our coding standard forward, perhaps a `constexpr int name = 10000000;` locally to this function? @OsiJr @peterAdams1014 thoughts on using `constexpr` for one offs or local values instead of `enum`?